app/i18n/nl/gen.php

```diff
@@ -79,6 +79,7 @@
 		'last_year' => 'Vorig jaar',
 		'mar' => 'mar',
```

Comment:
I'll fix this myself in a follow-up PR, but the Dutch abbreviation for `maart` is `mrt.` or `mrt`, not `mar`.